vagrant/src/main/java/org/jclouds/vagrant/internal/VagrantNodeRegistry.java

```diff
@@ -73,36 +93,41 @@ public long getDelay(TimeUnit unit) {
          return unit.convert(expiryTime - timeSupplier.get(), TimeUnit.MILLISECONDS);
       }
    }
-   private Map<String, VagrantNode> nodes = new ConcurrentHashMap<String, VagrantNode>();
-   private DelayQueue<TerminatedNode> terminatedNodes = new DelayQueue<TerminatedNode>();
+
+   private final DelayQueue<TerminatedNode> terminatedNodes = new DelayQueue<TerminatedNode>();
```

Comment:
Agree. Mind squashing the commits so I can merge it? Thanks for all explanations!